src/main/kotlin/no/fdk/dataset_catalog/service/DatasetService.kt

```diff
@@ -50,7 +50,7 @@ class DatasetService(
         val updatedAccessServiceUris: MutableSet<String> =
             accessServiceUris?.toMutableSet()
                 ?: mutableSetOf()
-        accessService?.mapNotNull { it.uri }
+        accessService?.mapNotNull { it.uri?.takeIf { uri -> uri.isNotBlank() } }
```

Comment:
Er det ikke bedre å fjerne alt som ikke er gyldig uri?
```suggestion
        accessService?.mapNotNull { it.uri }
            ?.filter { it.isValidURI() }
```